controllers/devfileregistrieslist_controller.go

```diff
@@ -67,12 +74,44 @@ func (r *DevfileRegistriesListReconciler) Reconcile(ctx context.Context, req ctr
 		return ctrl.Result{}, err
 	}
 
+	if devfileRegistriesList.Status.Conditions == nil || len(devfileRegistriesList.Status.Conditions) == 0 {
```

Comment:
Same comment as above for the other controller. Suggest using a more meaningful condition than `Reconciling`.

Reply:
@johnmcollier "Updating available devfile registry namespace list"?

Reply:
Addressed in https://github.com/devfile/registry-operator/pull/49/commits/33593bdd7b4a2b25c7bc3f5feafa69109841c45f